src/framework/preferences/js/SyllabificationEnactor.js

```diff
@@ -218,12 +223,14 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
         var hyphenator = fluid.getGlobalValue(globalPath);
 
         // If the pattern file has already been loaded, return the hyphenator.
+        // This could happen if the pattern file is statically linked to the page.
         if (hyphenator) {
             promise.resolve(hyphenator);
             return promise;
         }
 
         var src = fluid.stringTemplate(pattern, that.options.terms);
+
         var injectPromise = that.injectScript(src);
```

Comment:
This is a comment for the impl of injectScript - note that the return value of $.ajax is already a promise so there is no need to construct a further fluid.promise